layouts/error.vue

```diff
@@ -1,14 +1,67 @@
 <template>
-  <div>
-    <h1>{{ error }}</h1>
+  <div class="container bg-block">
+    <h1 v-if="error.statusCode === 404" class="main-text">
+      404
+      <p class="body-text">Woof... Sorry, this page does not exist!</p>
+    </h1>
+    <h1 v-else class="body-text">An error occurred - {{ error.statusCode }}</h1>
```

Comment:
```suggestion
    <p v-else class="is-size-3 has-text-weight-bold">An error occurred - {{ error.statusCode }}</p>
```
then delete the body-text class in the styles